### PR TITLE
Generated by startproject, to interact with pj

### PR DESCRIPTION
**Pull request by XiuqinCandice**
Dear@listonjesse,

I created the 1st project by following this [link](https://docs.djangoproject.com/en/3.0/intro/tutorial01/)

### CODE REVIEW

**Comment by XiuqinCandice** [comment]
👍 Sure, to be specific, Did you mean change the directory 'polls' to 'todo'?

**Comment by listonjesse** [comment]
> 👍 Sure, to be specific, Did you mean change the directory 'polls' to 'todo'?

I think the you should run 'python manage.py startapp todo' and delete the polls the directory. Renaming the directory would not be enough because there are files generated that contain the name [poll](https://github.com/XiuqinCandice/to-do-webapp/blob/1st-Django-app/basic_poll/mysite/polls/apps.py)

In addition to redoing the 'startapp' part, I think you should also rename the top directory from 'basic_poll' to 'mysite'

So in the end the directory structure will look like this:
mysite/
    manage.py
    mysite/
        __init__.py
        settings.py
        urls.py
        asgi.py
        wsgi.py
    todo/
        __init__.py
       admin.py
       apps.py
       migrations/
           __init__.py
       models.py
       tests.py
       views.py

**Comment by XiuqinCandice** [comment]
Thank you for the comments, I've done the following steps

- Moved the outer mysite folder to the parent level (basic_poll) and delete basic_poll.

- Restarted a new app to do and deleted polls

- Pointed the root URLconf at the todo.urls module
Can you help double-check the directory structure? Thanks a lot

File: mysite/todo/urls.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
```
Comment (listonjesse) [inline mysite/todo/urls.py]: nitpick: remove this new line

File: mysite/todo/views.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+from django.http import HttpResponse
+
```
Comment (listonjesse) [inline mysite/todo/views.py]: nitpick: put just one line between the imports and the first function definition

Comment (XiuqinCandice) [reply mysite/todo/views.py]: Thank you

File: mysite/todo/urls.py (hunk 1 above)
Comment (XiuqinCandice) [reply mysite/todo/urls.py]: Thanks

**Comment by XiuqinCandice** [comment]
Thank you @listonjesse I couldn't find .DS_Store file in the directory but I've added .DS_Store to .gitignore

**Comment by listonjesse** [comment]
@XiuqinCandice The .DS_Store file is a hidden file on mac, so you will need to run 'git rm .DS_Store' in a terminal to remove it since it won't show up in VSCode. Then when you remove it locally, you can commit the deletion

**Comment by listonjesse** [comment]
image

Don't forget to link the issues this PR solves (i.e. both Do Django tutorial and v1 web app issues) and also link our project so the board will update

Thanks!

**Comment by XiuqinCandice** [comment]
> @XiuqinCandice The .DS_Store file is a hidden file on mac, so you will need to run 'git rm .DS_Store' in a terminal to remove it since it won't show up in VSCode. Then when you remove it locally, you can commit the deletion

Thanks for your help @listonjesse . I've successful removed the file and committed it

**Comment by XiuqinCandice** [comment]
> image
> 
> Don't forget to link the issues this PR solves (i.e. both Do Django tutorial and v1 web app issues) and also link our project so the board will update
> 
> Thanks!

Do you mean this issue: "Set up repo with bare minimum web app using Django (Hello World) #1"?. I've linked it with this pull request, let me know. Thanks

**Comment by listonjesse** [comment]
> > image
> > Don't forget to link the issues this PR solves (i.e. both Do Django tutorial and v1 web app issues) and also link our project so the board will update
> > Thanks!
> 
> Do you mean this issue: "Set up repo with bare minimum web app using Django (Hello World) #1"?. I've linked it with this pull request, let me know. Thanks

Yes, looks good to me now. Thanks!

**Comment by listonjesse** [comment]
@XiuqinCandice Can you make sure to "Squash and Merge"? It will make our repo history look a lot tidier

image

**Comment by XiuqinCandice** [comment]
👍 💯 Will do, thank you @listonjesse.

**Comment by XiuqinCandice** [comment]
I found that even though I delete the branch, I can still see them in local, do I need to git branch -d 'branchname' to delete it and do git fetch -p to look for dead branches and remove those references at remote

**Pull request merged**